src/SFA.DAS.ApplyService.InternalApi/Services/CriminalComplianceChecksQuestionLookupService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.Domain.Roatp;
+using SFA.DAS.ApplyService.InternalApi.Models.Roatp;
+using System;
+
+namespace SFA.DAS.ApplyService.InternalApi.Services
+{
+    public class CriminalComplianceChecksQuestionLookupService : ICriminalComplianceChecksQuestionLookupService
+    {
+        public QnaQuestionDetails GetQuestionDetailsForGatewayPageId(string gatewayPageId)
+        {
+            switch (gatewayPageId)
```

Comment:
Whenever I see a switch I get nervous about how the code is going to end up.  As there is only 1 path I'd say this doesn't need a switch statement for now so we should remove it and look at alternative patterns as and when it grows.

Reply:
Currently only one, yes. The intent is for the same view/view model to be re-used for all the criminal compliance questions as they all have the same format. This is a lookup from gateway id to the QnA data it needs to lookup to populate the page